### PR TITLE
Added Training Program

### PR DESCRIPTION
**Pull request by Seized7x**
Included training program requested.

### CODE REVIEW

**Comment by glawson** [comment]
@Seized7x, please incorporate the script in main.js into the html, then delete main.js and resubmit.

File: training/michael.spencer.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+    <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.4.7/angular.min.js"></script>
+    <script src="myController.js"></script>
```
Comment (glawson) [inline training/michael.spencer.html]: The script from myController.js needs to be inline and that file myController.js deleted.

File: training/thibautHardin.html [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<html ng-app="">
+<script src= "http://ajax.googleapis.com/ajax/libs/angularjs/1.3.14/angular.min.js"></script>
+	<body>
+		<input id="in1" ng-model="textInput">
+		<h1>{{textInput}}</h1>
```
Comment (glawson) [inline training/thibautHardin.html]: No app or controller - try again.

**Comment by glawson** [comment]
@Seized7x, please have the team fix and then have them submit pull request individually.

**Comment by MSpencer87** [comment]
HW updated

**Comment by radfahrer** [comment]
This branch is out of date with master. Please merge upstream master into your branch.

**Comment by radfahrer** [comment]
@glawson has this been handled in another PR?

**Comment by glawson** [comment]
@Seized7x - please update this pull request.  If you are not sure how to, please email me directly.

Geoff

**Pull request closed without merging**